src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -33,20 +38,29 @@ def __init__(self, model, config_list):
         self.bound_model = model
         self.config_list = config_list
         self.modules_to_compress = []
+        self.detect_modules_to_compress()
 
-    def compress(self):
+    def detect_modules_to_compress(self):
         """
-        Compress the model with algorithm implemented by subclass.
+        detect all modules should be compressed, and save the result in `self.modules_to_compress`.
```

Comment:
I think it's better to return the result. Stateless is usually more readable.

Reply:
fixed